Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1779,4 +1779,16 @@ Import aborted.</value>
   <data name="EncapsulateField_WrapFieldsInPrivateType" xml:space="preserve">
     <value>Wrap Fields in Private Type</value>
   </data>
+  <data name="ExtractInterface_InstancingGroupBox" xml:space="preserve">
+    <value>Instancing</value>
+  </data>
+  <data name="ExtractInterface_Private" xml:space="preserve">
+    <value>Private</value>
+  </data>
+  <data name="ExtractInterface_Public" xml:space="preserve">
+    <value>Public</value>
+  </data>
+  <data name="ExtractInterface_PublicInstancingMandatedByPublicClass" xml:space="preserve">
+    <value>'Public' will be chosen. For private instancing make implementing class private also.</value>
```

Comment:
This needs a bit of reformulation, IMO. How about:

"Because the class the interface is extracted from is marked for public instancing, the interface must be public as well. If you want to extract a private interface, the class must be marked private before."